src/test/Profiling.cpp

```diff
@@ -171,7 +167,7 @@ TEST_F(ProfilingTest, Profiling_2) {
 			gameobject.add_component<BoxCollider>(vec2{0, 0}, vec2{1, 1});
 
 			gameobject.add_component<BehaviorScript>().set_script<TestScript>();
-			auto img = Asset("asset/texture/square.png");
+			auto img = Texture("asset/texture/square.png");
```

Comment:
Deze test is nu weer stuk. `Texture` moet denk ik nog vervangen worden met `Asset`.